Controllers/Mobile/v1/ClientsController.cs

```diff
@@ -17,8 +17,38 @@ namespace AonFreelancing.Controllers.Mobile.v1
     [ApiController]
     public class ClientsController(MainAppContext mainAppContext,
                                     ActivitiesService activitiesService,
-                                    UserService userService) : BaseController
+                                    UserService userService,
+                                    UserManager<User> userManager, AuthService authService, RoleManager<ApplicationRole> roleManager) : BaseController
     {
+ 
+
+
+        // before your review, you should i have skill issue here cuz the time is now 11:56 PM :)
```

Comment:
clean code and English are crying together in the corner now. 👀

Reply:
you should i have ?
i don't know how i wrote this :)